python/tk_syntheyes/ui/main_window.py

```diff
@@ -316,9 +368,18 @@ def generate_panels(self):
         self._main_panel.btn_context = self._main_panel.insert_menu_button(self._context_panel)
         self._main_panel.btn_context.setText(context_name)
         self._link_panel(self._main_panel.btn_context, self._context_panel)
-        self._main_panel.insert_line()        
+                
+        ######### DEBUG #########
+        from helper_functions import strtobool
+
+        if strtobool(os.environ.get("__DEV__", None)):
+            self.generate_dev_panel()
```

Comment:
Coole sache, ggf. nicht an der richtigen Stelle aber gute Idee. Bitte auch in der Doku bedenken damit wir das auch auf dem Schirm haben das es das gibt :)